.gitignore

```diff
@@ -1,2 +1 @@
-*.zip
```

Comment:
Is there a reason the .gitignore is changed? I’d rather we link to the zip-files used for the releases.

Reply:
I was hoping to keep it to just the PNG's instead of the full repo zip file.

I'm good with either though.

Reply:
The zip-files are hand crafted and only include the contents of the PNG folder + a link to this repo.

Reply:
If that fails I can host them on my personal domain.

Reply:
Ah, then that should be OK.